app/controllers/responsiblepeople/FitAndProperController.scala

```diff
@@ -102,7 +98,7 @@ class FitAndProperController @Inject()(
                                     msbOrTscp: Boolean,
                                     flow: Option[String])
                                    (implicit authContext: AuthContext, request: Request[AnyContent]): Result = {
-    (edit, fitAndProperAnswer, appConfig.phase2ChangesToggle) match {
+    (edit, fitAndProperAnswer, true) match {
```

Comment:
If the third item in the tuple is no longer a variable (I.e. always true); can we remove it?

Reply:
good spot, I'll change this